### DISCUSSION TITLE
How Access MySql to Extact Data

### DISCUSSION THREAD

**Discussion by javierllovet** [postalserver/postal#2252] [Help with using Postal]
I would like to know if anyone have a configuration or what steps i need to to in order to get access to mysql database and extract data to make anaylisis or review e-mail bad address are in suppresion list to correct part of our system that has incorrect e-mail address.

We would like to get access mysql from other computer or server to extract data and manage in our Power BI Server.

Anyone that can give us some guidance of this

**Comment by willpower232** [comment]
The problem is that Postal by default does not retain information forever so you would probably be better off analysing whatever sends messages instead.

**Reply by javierllovet** [reply]
I know is not retain information, at least i would like to access the data is that moment to play around, so do you know steps to access this from other host?

**Reply by willpower232** [reply]
It depends how you're running the database, you probably just need to open whatever firewall you're using but you should definitely be careful exposing a database to the internet like that so be selective where you open the firewall to.

It is also possible that your database user is limited locally so you may need to add another.
